server/chat-plugins/responder.ts

```diff
@@ -455,7 +456,7 @@ export const pages: PageTable = {
 				return `<h2>Invalid date.</h2>`;
 			}
 			buf = `<div class="pad"><strong>Stats for the ${room.title} auto-response filter${date ? ` on ${date}` : ''}.</strong>`;
-			buf += `${back}${refresh('stats', [date])}<hr />`;
+			buf += `${back}${refresh('stats', (date ? [date] : []))}<hr />`;
```

Comment:
is this necessary, since you are already filtering the array later

Reply:
Undone. Thank you!